platform/darwin/src/MGLRasterSource.mm

```diff
@@ -15,88 +17,77 @@ - (instancetype)initWithRawSource:(mbgl::style::RasterSource *)rawSource NS_DESI
 
 @end
 
-@implementation MGLRasterSource
-{
+@implementation MGLRasterSource {
     std::unique_ptr<mbgl::style::RasterSource> _pendingSource;
 }
 
-- (instancetype)initWithIdentifier:(NSString *)identifier URL:(NSURL *)url tileSize:(CGFloat)tileSize
-{
-    if (self = [super initWithIdentifier:identifier]) {
-        _URL = url;
-        _tileSize = tileSize;
-        [self commonInit];
-    }
-    return self;
+- (instancetype)initWithIdentifier:(NSString *)identifier configurationURL:(NSURL *)configurationURL {
+    return [self initWithIdentifier:identifier configurationURL:configurationURL tileSize:0];
```

Comment:
Continued in https://github.com/mapbox/mapbox-gl-native/pull/7377#discussion_r92281280.